src/Sulu/Bundle/WebsiteBundle/phpunit.xml.dist

```diff
@@ -1,33 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         bootstrap="../TestBundle/Resources/app/bootstrap.php"
-         colors="true"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
-    <coverage>
-        <include>
-            <directory>./</directory>
-        </include>
-
-        <exclude>
-            <directory>./Tests</directory>
-            <directory>./vendor</directory>
-        </exclude>
-    </coverage>
-
-    <testsuites>
-        <testsuite name="SuluWebsiteBundle Test Suite">
-            <directory suffix="Test.php">./Tests</directory>
-        </testsuite>
-    </testsuites>
-
-    <php>
-        <server name="APP_ENV" value="test" force="true"/>
-        <env name="KERNEL_CLASS" value="Sulu\Bundle\WebsiteBundle\Tests\Application\Kernel"/>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
-        <ini name="date.timezone" value="UTC"/>
-    </php>
-
-    <listeners>
-        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
-    </listeners>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="../TestBundle/Resources/app/bootstrap.php" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd" cacheDirectory=".phpunit.cache">
+  <testsuites>
```

Comment:
4 instead of 2 spaces for all our files like defined in the editorconfig: https://github.com/sulu/sulu/blob/d635d8dcaa8679c4ff5ab3f1f52a6fa7ea3bc53a/.editorconfig#L8